### PR TITLE
[policies] Add CloudLinux policy

### PR DESCRIPTION
**Pull request by eabdullin1**
CloudLinux OS is an Enterprise Linux distro, binary compatible with RHEL.
This commit adds a CloudLinux policy, which inherits the RedHatPolicy base.

Signed-off-by: Eduard Abdullin eabdullin@almalinux.org

---
Please place an 'X' inside each '[]' to confirm you adhere to our [Contributor Guidelines](https://github.com/sosreport/sos/wiki/Contribution-Guidelines)

- [X] Is the commit message split over multiple lines and hard-wrapped at 72 characters?
- [X] Is the subject and message clear and concise?
- [X] Does the subject start with **[plugin_name]** if submitting a plugin patch or a **[section_name]** if part of the core sosreport code?
- [X] Does the commit contain a **Signed-off-by: First Lastname <email@example.com>**?
- [X] Are any related Issues or existing PRs [properly referenced](https://docs.github.com/en/issues/tracking-your-work-with-issues/creating-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword) via a Closes (Issue) or Resolved (PR) line?
- [X] Are all passwords or private data gathered by this PR [obfuscated](https://github.com/sosreport/sos/wiki/How-to-Write-a-Plugin#how-to-prevent-collecting-passwords)?

### CODE REVIEW

**Review by TurboTurtle — APPROVED** [review]

File: sos/policies/distros/cloudlinux.py [hunk 1]
```diff
@@ -0,0 +1,46 @@
+# Copyright (C) Eduard Abdullin <eabdullin@cloudlinux.com>
```
Comment (pmoravec) [inline sos/policies/distros/cloudlinux.py:1]: Mail different from commit message is intentional? I expect it does not matter if both mails are correct.

**Comment by pmoravec** [comment]
No utilization of the policy for specifying e.g. own preset, child class for a `Plugin` to differentiate the Policy behaviour for a plugin or similar? Just a matter of proper "labeling" the sos run on that distro?

File: sos/policies/distros/cloudlinux.py (hunk 1 above)
Comment (eabdullin1) [reply sos/policies/distros/cloudlinux.py:1]: Yes, this was done on purpose

**Comment by eabdullin1** [comment]
> No utilization of the policy for specifying e.g. own preset, child class for a `Plugin` to differentiate the Policy behaviour for a plugin or similar? Just a matter of proper "labeling" the sos run on that distro?

Yes, for now, we don't have any special policies. This is only for "labeling"

File: sos/policies/distros/cloudlinux.py [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+# This file is part of the sos project: https://github.com/sosreport/sos
+#
+# This copyrighted material is made available to anyone wishing to use,
+# modify, copy, or redistribute it subject to the terms and conditions of
+# version 2 of the GNU General Public License.
```
Comment (pmoravec) [inline sos/policies/distros/cloudlinux.py:7]: Please add "..or later" per https://github.com/sosreport/sos/pull/3712

**Review by pmoravec — APPROVED** [review]

**Comment by eabdullin1** [comment]
@pmoravec Added "..or later" to the header

**Pull request merged**